diagnostics: 
  - id: 0x02
    fields:
      - name: "current"
        offset: 0
        len: 2
        is_signed: true
        factor: 0.01
        unit: "A"

  - id: 0x03
    fields:
      - name: "voltage"
        offset: 0
        len: 2
        is_signed: false
        factor: 0.01
        unit: "V"

  - id: 0x06
    fields:
      - name: "remaining_capacity"
        offset: 0
        len: 2
        is_signed: false
        factor: 0.001
        unit: "Ah"
      - name: "full_charge_capacity"
        offset: 2
        len: 2
        is_signed: false
        factor: 0.001
        unit: "Ah"

  - id: 0x04
    fields:
      - name: "temperature"
        offset: 0
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"

      - name: "temperature_cell_1"
        offset: 2
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"

      - name: "temperature_cell_2"
        offset: 4
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"

  - id: 0x05
    fields:
      - name: "temperature_cell_3"
        offset: 0
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"

      - name: "temperature_cell_4"
        offset: 2
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"

      - name: "temperature_cell_5"
        offset: 4
        len: 2
        is_signed: true
        factor: 0.1
        unit: "Celsius"


topics: ["current", "voltage", "remaining_capacity", "full_charge_capacity", "temperature"]

can_device: "can1"
bms_id_to_poll: 0x200
poll_frequency_hz: 20
